docker/nginx/conf.d/server/server.api

```diff
@@ -247,8 +247,7 @@ location /skynet/tus {
     proxy_set_header X-Forwarded-Proto $scheme;
 
     # rewrite proxy request to use correct host uri from env variable (required to return correct location header)
-    set_by_lua_block $server_domain { return os.getenv("SERVER_DOMAIN") }
```

Comment:
so I guess we can get rid of this because it's now set in the `server` blocks right?

Reply:
yes 👍